src/package.xml

```diff
@@ -1,126 +1,96 @@
-          <datasheets>
-            <datasheet name="outputTabularChange" displayName="Transitions Summary">
-              <columns>
-                <column name="outputTabularChangeID" dataType="Integer" isPrimary="True" />
-                <column name="ScenarioID" dataType="Integer" isCompositeIndex="True" />
-                <column name="movementTypesIDfrom" displayName="From" dataType="String" />
-                <column name="movementTypesIDto" displayName="To" dataType="String" />
-                <column name="amoutAreaChange" displayName="Area change" dataType="Double" validationType="Decimal" validationCondition="None" />
-                <column name="percentCoverChange" displayName="Proportion change" dataType="Double" validationType="Decimal" validationCondition="None" />
-              </columns>
-            </datasheet>
-          </datasheets>
-        </datafeed>
-        <datafeed name="outputTabularJaccard" displayName="Jaccard Dissimilarity" dataScope="Scenario">
-          <datasheets>
-            <datasheet name="outputTabularJaccard" displayName="Jaccard Dissimilarity">
-              <columns>
-                <column name="outputTabularJaccardID" dataType="Integer" isPrimary="True" />
-                <column name="ScenarioID" dataType="Integer" isCompositeIndex="True" />
-                <column name="movementTypesID" displayName="Connectivity category" dataType="Integer" validationType="Datasheet" formula1="omniscape_movementTypes" allowDbNull="False" />
-                <column name="dissimilarity" displayName="Dissimilarity" dataType="Double" validationType="Decimal" validationCondition="None" />
-              </columns>
-            </datasheet>
-          </datasheets>
-        </datafeed>
-      </datafeeds>
-    </transformer>
-  </transformers>
-  <layouts>
-    <layout name="coreforms_ScenarioDatafeeds" insertAfter="omniscape_Advanced">
-      <group name="Add0on" displayName="Add-on">
-        <group name="Impact0Assessment" displayName="Impact Assessment">
-          <item name="differenceScenarios" showIf="inPipeline" />
-          <group name="Results" displayName="Results">
-            <group name="Spatial" displayName="Spatial">
-              <item name="outputSpatialOverall" showIf="inPipeline" />
-              <item name="outputSpatialCategory" showIf="inPipeline" />
-            </group>
-            <group name="Tabular" displayName="Tabular">
-              <item name="outputTabularDifferences" showIf="inPipeline" />
-              <item name="outputTabularChange" showIf="inPipeline" />
-              <item name="outputTabularJaccard" showIf="inPipeline" />
-            </group>
+<package name="omniscapeImpact" displayName="Calculates changes in connectivity categories from a baseline" isAddOn="True" extendsPackage="omniscape" url="https://apexrms.github.io/omniscape/tutorials/omniscapeImpact" version="2.0.0">
```

Comment:
this package element is still syncrosim 2 - reference BurnP3+Prometheus/Cell2Fire to see what it should be